src/core/tsi/alts/handshaker/alts_handshaker_client.cc

```diff
@@ -298,7 +298,7 @@ void alts_handshaker_client_handle_response(alts_handshaker_client* c,
     if (details.size > 0) {
       error = absl::StrCat("Status ", code, " from handshaker service: ",
                            absl::string_view(details.data, details.size));
-      LOG(ERROR) << error;
+      LOG_EVERY_N(ERROR, 100) << error;
```

Comment:
While we're here, could we change this to an INFO level log? I don't think this should be an ERROR log.

Reply:
Done